index.bs

```diff
@@ -1197,18 +1192,39 @@ associated.
 </div>
 
 
+### RP Parameters for Credential Generation (dictionary <dfn dictionary>PublicKeyCredentialRpEntity</dfn>) ### {#sctn-rp-credential-params}
+
+The {{PublicKeyCredentialRpEntity}} dictionary is used to supply additional [=relying party=] attributes when creating a new credential.
+
+<pre class="idl">
+    dictionary PublicKeyCredentialRpEntity : PublicKeyCredentialEntity {
+        DOMString      id;
+    };
+</pre>
+
+<div dfn-type="dict-member" dfn-for="PublicKeyCredentialRpEntity">
+    :   <dfn>id</dfn>
+    ::  A unique identifier for the[=relying party=] entity, which sets the [=RP ID=].
+</div>
+
+
 ### User Account Parameters for Credential Generation (dictionary <dfn dictionary>PublicKeyCredentialUserEntity</dfn>) ### {#sctn-user-credential-params}
 
 The {{PublicKeyCredentialUserEntity}} dictionary is used to supply additional user account attributes when creating a new
 credential.
 
 <pre class="idl">
     dictionary PublicKeyCredentialUserEntity : PublicKeyCredentialEntity {
+        ArrayBuffer    id;
         DOMString      displayName;
     };
 </pre>
 
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialUserEntity">
+    :   <dfn>id</dfn>
+    ::  A unique identifier for the user account entity.
+        This is an opaque 32-byte array value specified by the [=relying party=].
```

Comment:
suggest: This is a reference to an opaque 32-byte value specified by the [=[RP]=].